spec/like_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Likes, type: :model do
```

Comment:
This is related to the suggestions highlighted above.

- [x] Please rename `Likes` to `Like`.

```suggestion
RSpec.describe Like, type: :model do
```